examples/src/main/java/io/dapr/examples/bindings/http/README.md

```diff
@@ -54,13 +54,13 @@ Run `dapr init` to initialize Dapr in Self-Hosted Mode if it's not already initi
 
 Before getting into the application code, follow these steps in order to set up a local instance of Kafka. This is needed for the local instances.
```

Comment:
Should this example also be deleted? its `bindings/http`

Reply:
Great question. I am keeping it because the client talks gRPC but the app's endpoint is HTTP. It is different example from when the app is serving gRPC.